partiql-planner/src/test/kotlin/org/partiql/planner/internal/typer/PlanTyperTestsPorted.kt

```diff
@@ -729,7 +729,7 @@ class PlanTyperTestsPorted {
             SuccessTestCase(
                 name = "BITWISE_AND_NULL_OPERAND",
                 query = "1 & NULL",
-                expected = StaticType.NULL,
+                expected = StaticType.unionOf(StaticType.INT4, StaticType.NULL),
```

Comment:
Why not StaticType.NULL?

Reply:
The value will be NULL, however, what is the type of the null value? I'm using our normal rules for type/function precedence to resolve the output types. See the following [PostgreSQL DB Fiddle Example](https://www.db-fiddle.com/f/jxMVQnmDXvRnc1tHCzhK96/0).